README.md

```diff
@@ -50,7 +50,7 @@ If you're developing a feature that requires the API, but can't or don't want to
 run it locally, you can specify `--env.api`:
 
 ``` sh
-yarn watch --env.api https://dev-api.va.gov
+yarn watch --env.api https://sandbox-api.va.gov
```

Comment:
Is this true for all of the Dev API? I thought it was only true for Lighthouse services? (with `services` as the first part of the path)

Reply:
The email notification didn't state it was only some URLs.  I'll post the email in slack as it might have private info in it.